lib/load.php

```diff
@@ -119,6 +119,7 @@ function gutenberg_is_experiment_enabled( $name ) {
 
 // WordPress 6.5 compat.
 require __DIR__ . '/compat/wordpress-6.5/block-patterns.php';
+require __DIR__ . '/compat/wordpress-6.5/class-wp-navigation-block-renderer.php';
```

Comment:
I don't think this is the right place for this class, but I'm not sure where it should go.